app/components/AlertDialog/__tests__/__snapshots__/AlertDialog.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AlertDialog It renders 1`] = `<DocumentFragment />`;
```

Comment:
I think I wouldn't bother with a snapshot in this case since this isn't including a snapshot of the dialog content anyway